client/dive-common/components/Viewer.vue

```diff
@@ -811,6 +813,10 @@ export default defineComponent({
       useAttributeFilters,
     );
 
+    function handleSelectTrack(trackId: number, modifiers: { ctrl: boolean } | null) {
+      handler.trackSelect(trackId, undefined, modifiers);
+    }
```

Comment:
I think you added this for the ControlsContainer, instead of doing this I believe you can directly import `handler = useHandler()` insid of the ControlsContainer.vue and use `handler.trackSelect`